### ISSUE TITLE
random seeds not considered anymore

### ISSUE THREAD

**Issue by chraibi** [JuPedSim/jpscore#209]
In Gitlab by @chraibi on May 25, 2016, 18:09 [[origin](https://gitlab.version.fz-juelich.de/jupedsim/jpscore/issues/195)]

Dear @all,

the recent major changes on JPScore develop resulted in a non-consideration of the random seeds. @laemmel (@OSchmidts) could you please have a look that immediately?   

This bug is likely to delay my work here in the US since I wanted to start the first simulation ensembles for ORPHEUS today. 

...appreciate more branch-focussed work...

**Comment by chraibi** [comment]
In Gitlab by @chraibi on Dec 17, 2016, 12:08

closed

**Comment by chraibi** [comment]
In Gitlab by @chraibi on Dec 17, 2016, 12:09

_Original comment by OSchmidts on 25 May 2016, 16:20_

---

you can see the history of commits here:
https://cst.version.fz-juelich.de/jupedsim/jpscore/commits/develop

if you know which commit broke the consideration of random seeds just do
```
git checkout <commit>
git checkout -b working_orpheus_branch
```

unfortunately i can't do more for you

**Comment by chraibi** [comment]
In Gitlab by @chraibi on Dec 17, 2016, 12:09

_Original comment by chraibi on 25 May 2016, 16:28_

---

I m on the edge of fixing this. Please wait

**Comment by chraibi** [comment]
In Gitlab by @chraibi on Dec 17, 2016, 12:09

_Original comment by chraibi on 26 May 2016, 10:15_

---

Somehow initialising the seed in `IniFileParser.cpp` (or even in `main.cpp`) has no effect on the shuffle function in `PedDistributer.cpp` 

Has someone an idea?

**Comment by chraibi** [comment]
In Gitlab by @chraibi on Dec 17, 2016, 12:09

_Original comment by chraibi on 26 May 2016, 11:39_

---

Fixed in 113f969b
